utils/utils.go

```diff
@@ -12,6 +11,8 @@ import (
 	"strings"
 	"syscall"
 
+	"golang.org/x/sys/unix"
```

Comment:
or you can see the use case on    line number 28..31 in this file (utils/utils.go)